src/v/cloud_storage/segment_meta_cstore.cc

```diff
@@ -552,7 +553,9 @@ class column_store
         auto bo = *base_offset_iter;
         auto ix = base_offset_iter.index();
         auto hint_it = _hints.lower_bound(bo);
```

Comment:
Should we not do this lower_bound if hints are ignored?

Reply:
makes sense, made the change